wp/lib/bap_wp/src/environment.ml

```diff
@@ -42,7 +44,10 @@ type t = {
   loop_handler : loop_handler;
   exp_conds : exp_cond list;
   arch : Arch.t;
-  fun_input_regs : bool
+  fun_input_regs : bool;
+  stack : Constr.z3_expr -> Constr.z3_expr;
```

Comment:
What's the input to this feller? Comments here might be useful.

Reply:
Can do.